third_party/jsign/.gitignore

```diff
@@ -0,0 +1 @@
+*.jar
```

Comment:
You should update the clean script as well.

Reply:
added clean step to `package.json`, but i feel this somehow separates the logic. Ideally we should have an `clean.action.mjs` which uses the `clean()` method defined in `third_party/jsign/index.mjs`.